doc/tutorials/namespaces.md

```diff
@@ -99,74 +100,104 @@ Having namespaces enables both admins and users to set specific parts of the app
 
 ## How it Works on the Command Line (kdb) ##
 
-Let's say your app requires the following configuration data:
+We will provide an example of how you can configure [elektrified](../help/elektra-glossary.md) applications.
```

Comment:
I am not sure if website+doxygen works with relative links. Let us wait for the build jobs.